### PR TITLE
done with makeModel()

### PR DESCRIPTION
**Pull request by Priyanka-Kasireddy**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -25,7 +25,16 @@
 Returns: None
 '''
 def makeModel(data):
-    return
+    data["no.of rows"]=10
```
Comment (manasa133) [inline battleship.py:28]: variable names should be changed, it should be _ instead of dots. pls change.

File: battleship.py [hunk 2]
```diff
@@ -25,7 +25,16 @@
 Returns: None
 '''
 def makeModel(data):
-    return
+    data["no.of rows"]=10
+    data["no.of cols"]=10
+    data["board size"]=500
```
Comment (manasa133) [inline battleship.py:30]: instead of space use upper case character or underscore as a separator

**Review by manasa133 — CHANGES_REQUESTED** [review]
Change the mentioned comments

File: battleship.py (hunk 1 above)
Comment (Priyanka-Kasireddy) [reply battleship.py:28]: changed mam

File: battleship.py (hunk 2 above)
Comment (Priyanka-Kasireddy) [reply battleship.py:30]: changed mam

**Comment by Priyanka-Kasireddy** [comment]
done with the changes mam

**Pull request merged**